pkg/webhooks/clusterqueue_webhook.go

```diff
@@ -144,6 +145,15 @@ func validatePreemption(preemption *kueue.ClusterQueuePreemption, path *field.Pa
 	return allErrs
 }
 
+func validateCQAdmissionChecks(spec *kueue.ClusterQueueSpec, path *field.Path) field.ErrorList {
```

Comment:
Could you implement unit tests here?: https://github.com/kubernetes-sigs/kueue/blob/b9c346a83d965d7849e4e27cf7e06e5ee1ad22df/pkg/webhooks/clusterqueue_webhook_test.go#L34

Reply:
Added